src/Screens/Explore/components/DiveSite.tsx

```diff
@@ -82,7 +90,8 @@ const DiveSite: FunctionComponent<DiveSiteProps> = props => {
       <TouchableOpacity
         activeOpacity={0.8}
         onPress={() =>
-          props.onPressContainer && props.onPressContainer(props.site.id)
+          props.onPressContainer &&
```

Comment:
nit: i think we can delete this since it's already checked above

Reply:
Tried before. Compiler bleated. Decided to leave it alone. Complier is still bleating.

Reply:
oh weird. that's fine then